docs/extend/plugins_network.md

```diff
@@ -11,30 +11,28 @@ weight=-1
 
 # Docker network driver plugins
 
-Docker supports network driver plugins via
-[LibNetwork](https://github.com/docker/libnetwork). Network driver plugins are
-implemented as "remote drivers" for LibNetwork, which shares plugin
-infrastructure with Docker. In effect this means that network driver plugins
-are activated in the same way as other plugins, and use the same kind of
-protocol.
+Docker supports network driver plugins via LibNetwork project. Network driver
```

Comment:
s/via LibNetwork project/via *the* LibNetwork project/ ?

(Note: I know it's WIP, just reading through, in case you missed some)